examples/choropleth-viz-example.ipynb

```diff
@@ -90,6 +92,152 @@
+    "\n",
+    "viz = AltChoroplethViz(None, \n",
+    "                       vector_polygon_source=True,\n",
+    "                       vector_url='mapbox://mapbox.us_census_states_2015',\n",
+    "                       vector_layer_name='states',\n",
+    "                       vector_join_color_property='STATEFP',\n",
+    "                       join_data=join_data,\n",
+    "                       data_join_property='id',\n",
+    "                       color_property='density',\n",
+    "                       color_stops=sample_color_stops,\n",
+    "                       line_stroke='dashed',\n",
+    "                       line_color='rgb(128,0,38)',\n",
+    "                       center=(-96, 37.8),\n",
+    "                       zoom=3,\n",
+    "                       below_layer='waterway-label'\n",
+    "                    )\n",
+    "viz.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pre-computed colors for styling polygons in vector layer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sample_color_stops = [\n",
+    "    [0.0, 'rgb(255,255,204)'],\n",
+    "    [50.0, 'rgb(255,237,160)'],\n",
+    "    [100.0, 'rgb(253,141,60)'],\n",
+    "    [500.0, 'rgb(227,26,28)'],\n",
+    "    [2500.0, 'rgb(189,0,38)'],\n",
+    "    [5000.0,'rgb(128,0,38)']\n",
+    "]\n",
+    "\n",
+    "# some pre-computed colors (avoid computing colors in JavaScript template)\n",
```

Comment:
Once we have the class finalized, let's generate this data-join color stops with a helper function instad of manually.